app/domain/authentication/authn_jwt/authentication_parameters.rb

```diff
@@ -2,7 +2,7 @@ module Authentication
   module AuthnJwt
     # Data class to store data regarding jwt token that is needed during the jwt authentication process
     class AuthenticationParameters
-      attr_accessor :jwt_identity, :decoded_token, :jwt_token
+      attr_accessor :decoded_token, :jwt_token
```

Comment:
does `jwt_token` need to be an `attr_accessor`? i searched for `.jwt_token` in the code and couldn't find anything.

Reply:
We need it in validate_and_decode . There is deligator needing it as accessor.
I tried removed each of the accessors there. Only the identity can be removed in peace.

Reply:
for now :)